code/game/objects/items/devices/versebook.dm

```diff
@@ -1,5 +1,6 @@
 /obj/item/device/versebook
 	name = "\improper versebook"
+	w_class = ITEMSIZE_SMALL
```

Comment:
consider adding this under `item_state = "dominia"` to be in line with how variables on objects are usually lined up.